### PR TITLE
Adding a windows batch file to act like the stage1 for other environm…

### PR DESCRIPTION
**Pull request by Brodiez**
…ents, updated readme to include the new batch file options

Signed-off-by: Brodie <bzoschke@gmail.com>

Contributes to # (add issue number here. i.e. `#302`)

## What did you do?
created a windows batch file to act like the current stage1 file for other environments
## Why did you do it?
to simplify the process if a developer is using windows
## How have you tested it?
Ran all the commands through the script and matched them to calling the steps directly
## Were docs updated if needed?

- [ ] No
- [x] Yes
- [ ] N/A

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new console logs
- [ ] Fixes entire issue
- [ ] Partial fix for issue

### CODE REVIEW

File: stage1.bat [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+::
+:: Examples:
+:: stage1.bat runserver localhost:3000   <--- default if nothing specified
+:: stage1.bat shell
+:: stage1.bat makemigrations
+:: stage1.bat migrate
+:: stage1.bat dbshell
+:: stage1.bat qcluster
+SET parms="%*"
+if [%parms%] == [""] ( SET parms=runserver localhost:3000 ) ELSE ( SET parms=%parms:"=%)
```
Comment (Brodiez) [inline stage1.bat:12]: The else strips quotes from the provided parameters

**Comment by upkarlidder** [comment]
@Brodiez the bash script also sets the USE_SQLITE3 environment variable. Can we add that to this script as well?

```
USE_SQLITE3='True'
```

FYI @tonypearson @demilolu

**Comment by Brodiez** [comment]
> @Brodiez the bash script also sets the USE_SQLITE3 environment variable. Can we add that to this script as well?
> 
> ```
> USE_SQLITE3='True'
> ```
> 
> FYI @tonypearson @demilolu

done

**Review by upkarlidder — APPROVED** [review]
Thank you for making the changes as requested.

**Pull request merged**